scripts/converter/convert_eun_to_nue.py

```diff
@@ -38,11 +38,19 @@ def convert_to_nue(filename):
 
     for node in world.content['root']:
         if node['name'] == 'WorldInfo':
+            is_nue = False
             for field in node['fields']:
                 if field['name'] == 'northDirection':
                     assert field['value'] == ['0', '0', '1']
                     # remove the 'northDirection 0 0 1'
                     del node['fields'][node['fields'].index(field)]
+                if field['name'] == 'coordinateSystem':
+                    field['value'] == 'NUE'
```

Comment:
Is this a comparison or an assignment?
Maybe it should be
```suggestion
                    field['value'] = 'NUE'
```

Reply:
😨 you're right well spotted, I am surprised it worked when I tested it.